test/mocha/27-http-edv-client-update.js

```diff
@@ -119,5 +119,22 @@ describe('bedrock-edv-storage HTTP API - edv-client update', () => {
         err.message.should.equal(
           '"sequence" is too large.');
       });
+    it('should increase sequence when updating a deleted document',
+      async () => {
+        await edvClient.insert({
+          doc: mockData.httpDocs.alpha,
+          invocationSigner: capabilityAgent.getSigner(),
+        });
+        // delete doc
+        await edvClient.delete({id: mockData.httpDocs.alpha.id,
+          invocationSigner: capabilityAgent.getSigner()});
```

Comment:
Stylistically, if we move any parameter down we move the first down as well to keep parameters together in the same visual block/structure/pattern.

```suggestion
        await edvClient.delete({
          id: mockData.httpDocs.alpha.id,
          invocationSigner: capabilityAgent.getSigner()
        });
```

Reply:
Updated.